junit-platform-launcher/src/main/java/org/junit/platform/launcher/MethodFilter.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2015-2024 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.launcher;
```

Comment:
Please move this interface and its implementations to the `org.junit.platform.engine.discovery` package for consistency with `ClassNameFilter`.

Reply:
Currently, `MethodFilter` extends `PostDiscoveryFilter`. If I move it to the `org.junit.platform.engine.discovery` package, should I update it to `MethodFilter extends DiscoveryFilter<String>`, similar to `ClassNameFilter`?

Reply:
Thanks for pointing that out! I had missed that difference when reviewing the PR. If we changed it to `DiscoveryFilter`, we would be relying on each engine to apply it correctly during discovery (if it's not using `EngineDiscoveryRequestResolver` where we could make that change). Thus, I think we should stick with `PostDiscoveryFilter`. Since `TagFilter` is in this package already, let's keep this new filter here as well.